packages/synapse-react-client/src/lib/containers/table/TopLevelControls.tsx

```diff
@@ -221,6 +229,17 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
                 </button>
               )
             })}
+          {showExportToCavatica && (
+            <Button
+              variant="outlined"
+              onClick={() => {
+                exportToCavatica()
+              }}
+            >
+              <Cavatica />
+              &nbsp;&nbsp;Send to Cavatica
```

Comment:
If you used MUI SvgIcon as suggested above, you could add margin instead of using nbsp
```suggestion
              <Cavatica sx={{mr: 1}} />
              Send to Cavatica
```